assignments/lambda-classes.js

```diff
@@ -1 +1,139 @@
 // CODE here for your Lambda Classes
+class Person
+{
+    constructor(personalInformation)
+    {
+        this.name = personalInformation.name;
+        this.age = personalInformation.age;
+        this.location = personalInformation.location;
+        this.gender = personalInformation.gender;
+    }
+    speak()
+    {
+        console.log(`Hello my name is ${this.name}, I am from ${this.location}`);
+    }
+}
+
+class Instructor extends Person
+{
+    constructor(instructorInformation)
+    {
+        super(instructorInformation);
+        this.specialty = instructorInformation.specialty;
+        this.favLanguage = instructorInformation.favLanguage;
+        this.catchPhrase = instructorInformation.catchPhrase;
+    }
+    demo(subject)
+    {
+        console.log(`Today we are learning about ${subject}`);
+    }
+    grade(student, subject)
+    {
+        console.log(`${student.name} receives a perfect score on ${subject}`);
+    }
+    randomPoints(student)
+    {
+        let randomNumber = Math.floor(Math.random()*10)-3;
+        student.grade -= randomNumber;
+    }
+}
+
+class Student extends Person
+{
+    constructor(studentInformation)
+    {
+        super(studentInformation);
+        this.previousBackground = studentInformation.previousBackground;
+        this.className = studentInformation.className;
+        this.favSubjects = studentInformation.favSubjects;
+        this.grade = 50;
+    }
+    listsSubjects()
+    {
+        this.favSubjects.forEach(subject => console.log(subject));
+    }
+    PRAssignment(subject)
+    {
+        console.log(`${this.name} has begun sprint challenge on ${subject}`);
+    }
+    graduate()
+    {
+        if(this.grade>70)
+        {
+            console.log(`${this.name} is ready to graduate.`);
+        }
+        else
+        {
+            console.log(`Go back to grading ${this.name}'s assignments.`);
+        }
+    }
+}
+
+class ProjectManager extends Instructor
+{
+    constructor(pMInfo)
+    {
+        super(pMInfo);
+        this.gradClassName = this.pMInfo;
```

Comment:
Check this constructor against the other two.... one does not look like the others..